config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
We should probably exclude this file from git.

Reply:
Fixed - I removed my changes. It is in the gitignore but this file has always been odd